### ISSUE TITLE
Monorepo issue with angular.json

### ISSUE THREAD

**Issue by Zankel-Engineering** [nxtend-team/nxtend#615]
Hello gentleman,

I currently face an issue in my nx monorepo with an Ionic 6 project and your library. As soon as I run:

```nx run text-macht-mobile-app:add:ios````

He complains about the version in the angular.json which is 2 at the moment and he expects 1. The compiler says:

Unable to read workspace file '.../angular.json': Invalid format version detected - Expected:[ 1 ] Found: [ 2 ]

When I change the version nothing works. Any ideas how to fix that?

Help would be greatly appreciated.

Thanks,

Sebastian

**Comment by Zankel-Engineering** [comment]
No comments?

**Comment by Zankel-Engineering** [comment]
Not possible to set that plugin up working with nx workspace and Angular 13 👎 👎 👎 . Outdated.

**Comment by mrambow** [comment]
Having the same issue :)

the incompatibility isn't with angular 13 or nx 13, it's with the nx standalone config..
as described here: https://nx.dev/getting-started/nx-and-angular#angularjson -- don't do that 😅

angular, ionic, capacitor and nearly all other plugins require a single `angular.json` (or `workspace.json`), with the standalone config there isn't any and nx / nxtend need to wrap the schematic to generate a single `angular.json`.. 

you could wait until it's wrapped correctly or just go back to one single `angular.json` to rule them all ;)

**Comment by Zankel-Engineering** [comment]
Nx 13 only works with schema version 2 isn't it? They even have a schmatic to format with ```nx format```.

**Comment by mrambow** [comment]
Working even without version 2, but I only used build and run with 13 🤔

**Comment by Zankel-Engineering** [comment]
What version is written in your angular.json? Does your project have project.json files for the apps and libs? Cause then it only works with schema version 2 in the angular.json, as far as I was able to see. I recreated the whole workspace to try to make it run, but ran into plenty dependency issues, when I go back to nx version 12.x.

**Comment by mrambow** [comment]
You're right, the standalone config (project.json in every lib/app) only works with angular.json version 2, as described by NX in the link I shared.

But you can just use the old angular.json(version 1 with all config in the root angular json) with NX 13.

**Comment by Zankel-Engineering** [comment]
Hi man but I tried to move everything from project.json's into angular.json and changed the version and ran into plenty of errors. Can you explain what I did wrong? Thanks

**Comment by Markus-Ende** [comment]
> No comments? Why you don't mention that it is not compatible with newest version of NX 👎 ! I have to set up the whole workspace now again!

No you don't. You could also just don't use it or create a PR to fix. It's open source.
